README.md

```diff
@@ -135,6 +135,7 @@ See also [Rust — Production](https://www.rust-lang.org/production) organizatio
 * [rx](https://github.com/cloudhead/rx) — Vi inspired Modern Pixel Art Editor
 * [tiny](https://github.com/osa1/tiny) — A terminal IRC client
 * [updns](https://github.com/wyhaya/updns) — DNS proxy tool
+* [wezterm](https://github.com/wez/wezterm) — A GPU-accelerated cross-platform terminal emulator and multiplexer
```

Comment:
As a general rule, for projects (certainly newly added ones) that have a CI system, we like to add the build badge image for that project to the end of the line so people can see the project is functioning. Wezterm has a Github Actions build, so we want the badge from that. https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#adding-a-workflow-status-badge-to-your-repository details how to get the markup for that, and we require that you use the branch filtering to filter down to just the main branch for the project (usually "master").

Reply:
@palfrey The github actions page for that project lists about 45 builds. https://github.com/wez/wezterm/actions , one for each OS (Fedora, macOS, Windows) etc. I don't know how to proceed with this next?

Reply:
Wow, that's a lot of builds. And annoyingly Github actions doesn't allow you to have a "did everything for this branch work" badge :( Right, I think this will have to get merged as-is then.